scripts/regression/test_aerodynamics.py

```diff
@@ -214,32 +211,91 @@ def reg_values():
            cd_m_nac_base_r[:,None], cd_m_ctrl_r[:,None], cd_p_fuse_r[:,None], cd_p_wing_r[:,None], cd_tot_r[:,None]
 
 if __name__ == '__main__':
-    (conditions, configuration, geometry, test_num) = main()
+    #(conditions, configuration, geometry, test_num) = main()
+    main()
     
     print 'Aero regression test passed!'
     
+    ## --------------------------------------------------------------------
+    ## Drag Polar
+    ## --------------------------------------------------------------------  
+    
     # --------------------------------------------------------------------
     # Drag Polar
-    # --------------------------------------------------------------------  
+    # --------------------------------------------------------------------
     
-    # Cruise conditions (except Mach number)
-    conditions.freestream.mach_number = np.array([0.2]*test_num)
-    conditions.freestream.density = np.array([0.3804534]*test_num)
-    conditions.freestream.viscosity = np.array([1.43408227e-05]*test_num)
-    conditions.freestream.temperature = np.array([218.92391647]*test_num)
-    conditions.freestream.pressure = np.array([23908.73408391]*test_num)
+    # initialize the vehicle
```

Comment:
this block (if name==main) should stay clean, the plotting code can go in main(), just don't call plt.show() in main.